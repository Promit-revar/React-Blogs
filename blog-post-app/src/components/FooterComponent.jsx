import React from 'react';
import '../assets/style.css';
export default function FooterComponent() {
    return (
        <footer className='footer '>
            <p>@Artifact.com 2021</p>
        </footer>
    );  
}