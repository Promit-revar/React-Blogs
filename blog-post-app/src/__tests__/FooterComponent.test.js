import FooterComponent from '../components/FooterComponent';
import { render } from '@testing-library/react';
import React from 'react';
describe('Footer Snapshot', () => {
  it('should render the Footer component', () => {
    const {asFragment} = render(<FooterComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
