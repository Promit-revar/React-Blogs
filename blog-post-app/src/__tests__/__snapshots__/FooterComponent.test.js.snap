// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot should render the Footer component 1`] = `
<DocumentFragment>
  <footer
    class="footer "
  >
    <p>
      @Artifact.com 2021
    </p>
  </footer>
</DocumentFragment>
`;
